### ISSUE TITLE
Differential color refraction correction

### ISSUE THREAD

**Issue by dfarnocchia** [IAU-ADES/ADES-Master#13]
When observing through the atmosphere, the apparent position of an object is biased because of differential color refraction.
The bias is a function of the color of the observed target (which is generally unknown) and that of the stars used for the astrometric solution.
The bias is larger at lower elevation.

This term is to be computed by observers, when feasible.
ADES should have a note to specify that the observation was in fact color corrected and the correction should be reported in the remarks field.

**Comment by stevechesley** [comment]
Agreed. I guess we need a new 1-character note from the MPC for this one? Will leave this open until I get a chance to update the documentation.

**Comment by federicaspoto** [comment]
I added a new note `L` for color corrected observations. The new note is available here: `https://minorplanetcenter.net/mpcops/documentation/valid-ades-values/#notes`

**Comment by stevechesley** [comment]
Thanks, but I'm doubtful that the description "Color corrected observations" will be clear to most users. it sounds like a photometric correction. It may be better to say "Corrected for differential color refraction" or something like that.
